tfupdate/module.go

```diff
@@ -51,12 +53,24 @@ func (u *ModuleUpdater) Update(_ context.Context, _ *ModuleContext, filename str
 	return u.updateModuleBlock(f)
 }
 
+func match(match, matchType, name string) bool {
+	switch matchType {
+	case "regex":
+		var re = regexp.MustCompile(match)
```

Comment:
Compiling the regular expression for each comparison is expensive, so compiling it only once when parsing the option is better.
Furthermore, MustCompile panics on failure, but since the regular expression is an arbitrary string given by users, it should return an error instead of panic.

Reply:
This is fixed with the commit [e1441c9a5653fb2a577701c7e6000539839dd874].(https://github.com/minamijoyo/tfupdate/pull/108/commits/e1441c9a5653fb2a577701c7e6000539839dd874)